pkg/riff/commands/application_create.go

```diff
@@ -142,6 +146,22 @@ func (opts *ApplicationCreateOptions) Exec(ctx context.Context, c *cli.Config) e
 		return err
 	}
 	c.Successf("Created application %q\n", application.Name)
+	if opts.Tail {
+		// cancel ctx when application becomes ready
+		ctx, cancel := context.WithCancel(ctx)
+		go func() {
+			defer cancel()
+			applicationWatch, err := c.Build().Applications(opts.Namespace).Watch(metav1.ListOptions{
+				ResourceVersion: application.ResourceVersion,
+			})
+			if err != nil {
+				return
```

Comment:
Unless we know the error has already been logged, it would be good to log it before returning to provide some visibility into what went wrong.

Reply:
done